nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -124,17 +124,25 @@ Costmap2DROS::Costmap2DROS(const std::string & name, tf2_ros::Buffer & tf)
   setPluginParams(node_);
   }
 
-  if (parameters_client_->has_parameter("plugin_names") &&
-    parameters_client_->has_parameter("plugin_types")) {
-    auto param = get_parameters({"plugin_names", "plugin_types"});
-    for (int32_t i = 0; i < param[0].get_value<std::vector<std::string>>().size(); ++i) {
-      std::string pname = (param[0].get_value<std::vector<std::string>>())[i];
-      std::string type = (param[1].get_value<std::vector<std::string>>())[i];
-      RCLCPP_INFO(get_logger(), "Using plugin \"%s\"", pname.c_str());
-      std::shared_ptr<Layer> plugin = plugin_loader_.createSharedInstance(type);
-      layered_costmap_->addPlugin(plugin);
-      plugin->initialize(layered_costmap_, name + "_" + pname, &tf_, node_);
-    }
+  // if (parameters_client_->has_parameter("plugin_names") &&
```

Comment:
Should this code be removed? Otherwise is there an issue for getting it working?

Reply:
In general, I'd like to remove commented-out code unless it will be enabled relatively soon.

Reply:
This code is a temporary workaround. I expect to delete my addition and re-enable the commented out code very soon. We just need to figure out what the right way to do it is. There is a race condition of some sort between where the parameters are set and here (where we read the parameters).

Reply:
Created issue #262 to fix this